<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1</width>
      <height>1</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>-50</x>
       <y>0</y>
       <width>800</width>
       <height>600</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_File">
     <property name="title">
      <string>Recent File</string>
     </property>
     <addaction name="actionFlie_List"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent_File"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_Image"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave_All"/>
    <addaction name="separator"/>
    <addaction name="actionImport_Data"/>
    <addaction name="actionOutput_Data"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuFind_or_Replace">
     <property name="title">
      <string>Find or Replace</string>
     </property>
     <addaction name="actionFind"/>
     <addaction name="actionReplace"/>
    </widget>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="menuFind_or_Replace"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuDraw_Tools">
     <property name="title">
      <string>Draw Tools</string>
     </property>
     <widget class="QMenu" name="menuLine">
      <property name="title">
       <string>Line</string>
      </property>
      <addaction name="actionArc"/>
      <addaction name="actionStraight_Line"/>
     </widget>
     <addaction name="menuLine"/>
     <addaction name="actionCircle"/>
     <addaction name="actionRectangle"/>
    </widget>
    <addaction name="menuDraw_Tools"/>
    <addaction name="separator"/>
    <addaction name="actionPen_Thickness"/>
    <addaction name="actionPen_Color"/>
    <addaction name="actionBackground_Color"/>
    <addaction name="actionClues_Color"/>
    <addaction name="actionClues_Thickness"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuNodes_Edges_Details">
     <property name="title">
      <string>Nodes and Edges Details</string>
     </property>
     <widget class="QMenu" name="menuNode_s_Degree">
      <property name="title">
       <string>Node's Degree</string>
      </property>
      <widget class="QMenu" name="menuDigraph_s_Degrees">
       <property name="title">
        <string>Digraph's Degrees</string>
       </property>
       <addaction name="actionOut_degree"/>
       <addaction name="actionIn_degree"/>
      </widget>
      <addaction name="menuDigraph_s_Degrees"/>
      <addaction name="actionRedigraph_s_Degrees"/>
     </widget>
     <addaction name="actionNode_s_Name"/>
     <addaction name="actionNode_s_Coordinates"/>
     <addaction name="actionEdge_s_Weight"/>
     <addaction name="menuNode_s_Degree"/>
    </widget>
    <widget class="QMenu" name="menuPathway_and_Loop">
     <property name="title">
      <string>Pathway and Loop</string>
     </property>
     <widget class="QMenu" name="menuPathway">
      <property name="title">
       <string>Pathway</string>
      </property>
      <addaction name="actionEasy_Pathway"/>
      <addaction name="actionComplicated_Pathway"/>
     </widget>
     <widget class="QMenu" name="menuLoop">
      <property name="title">
       <string>Loop</string>
      </property>
      <addaction name="actionEasy_Loop"/>
      <addaction name="actionComplicated_Loop"/>
     </widget>
     <addaction name="menuPathway"/>
     <addaction name="menuLoop"/>
    </widget>
    <widget class="QMenu" name="menuGraph_Matrix">
     <property name="title">
      <string>Graph Matrix</string>
     </property>
     <addaction name="actionIncidence_Matrix"/>
     <addaction name="actionAdjacent_Matrix"/>
    </widget>
    <widget class="QMenu" name="menuSpecial_Graph">
     <property name="title">
      <string>Special Graph</string>
     </property>
     <widget class="QMenu" name="menuEuler_Graph">
      <property name="title">
       <string>Euler Graph</string>
      </property>
      <addaction name="actionFind_Euler_Loop"/>
     </widget>
     <widget class="QMenu" name="menuHamiltonian_Graph">
      <property name="title">
       <string>Hamiltonian Graph</string>
      </property>
      <addaction name="actionFind_Hamiltonian_Loop"/>
     </widget>
     <addaction name="actionBigraph"/>
     <addaction name="menuEuler_Graph"/>
     <addaction name="menuHamiltonian_Graph"/>
    </widget>
    <widget class="QMenu" name="menuShortest_Path_and_Critical_Path">
     <property name="title">
      <string>Shortest Path and Critical Path</string>
     </property>
     <addaction name="actionShortest_Path"/>
     <addaction name="actionCritical_Path"/>
    </widget>
    <addaction name="menuNodes_Edges_Details"/>
    <addaction name="menuPathway_and_Loop"/>
    <addaction name="menuGraph_Matrix"/>
    <addaction name="menuSpecial_Graph"/>
    <addaction name="menuShortest_Path_and_Critical_Path"/>
    <addaction name="actionGraph_Coloring_Problem"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <widget class="QMenu" name="menuGraph_Mode">
     <property name="title">
      <string>Graph Mode</string>
     </property>
     <addaction name="actionDigraph_Mode"/>
     <addaction name="actionRedigraph_Mode"/>
    </widget>
    <addaction name="menuGraph_Mode"/>
    <addaction name="actionTree_Mode"/>
   </widget>
   <widget class="QMenu" name="menuWinodws">
    <property name="title">
     <string>Winodws</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
     <addaction name="actionSetEnglish"/>
     <addaction name="actionSetChinese"/>
    </widget>
    <addaction name="actionMinimise"/>
    <addaction name="actionMaximize"/>
    <addaction name="actionMini_Mode"/>
    <addaction name="actionHide_ToolBar"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionConnect_Author"/>
    <addaction name="actionWelcome"/>
    <addaction name="actionHelp_Document"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSettings"/>
   <addaction name="menuTools"/>
   <addaction name="menuMode"/>
   <addaction name="menuWinodws"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="general_utility_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionPen_Color"/>
   <addaction name="actionPen_Thickness"/>
   <addaction name="actionBackground_Color"/>
   <addaction name="actionClues_Color"/>
   <addaction name="actionClues_Thickness"/>
  </widget>
  <widget class="QToolBar" name="graphical_toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionArc"/>
   <addaction name="actionStraight_Line"/>
   <addaction name="actionCircle"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionAdd_Annotation"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>251</width>
       <height>621</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <widget class="QUndoView" name="undoView">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>241</width>
            <height>591</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <widget class="QListView" name="listView">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>10</y>
            <width>241</width>
            <height>581</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget_2">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/128.bmp</normaloff>:/icons/images/128.bmp</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="toolTip">
    <string>新建文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/132.bmp</normaloff>:/icons/images/132.bmp</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="toolTip">
    <string>打开文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/SaveFile.png</normaloff>:/icons/images/SaveFile.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>保存文件</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="actionSave_All">
   <property name="text">
    <string>Save All</string>
   </property>
   <property name="toolTip">
    <string>保存全部</string>
   </property>
  </action>
  <action name="actionImport_Data">
   <property name="text">
    <string>Import Data</string>
   </property>
   <property name="toolTip">
    <string>导入数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionOutput_Data">
   <property name="text">
    <string>Output Data</string>
   </property>
   <property name="toolTip">
    <string>导出数据</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionSave_Image">
   <property name="text">
    <string>Save Image</string>
   </property>
   <property name="toolTip">
    <string>保存为图片</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/108.bmp</normaloff>:/icons/images/108.bmp</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="toolTip">
    <string>关闭文件</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/DELETE.BMP</normaloff>:/icons/images/DELETE.BMP</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>退出应用</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/undo.png</normaloff>:/icons/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/redo.png</normaloff>:/icons/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>剪贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/DELETE.BMP</normaloff>:/icons/images/DELETE.BMP</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="toolTip">
    <string>查找</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionReplace">
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="toolTip">
    <string>替换</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionNode_s_Name">
   <property name="text">
    <string>Node's Name</string>
   </property>
   <property name="toolTip">
    <string>结点名称</string>
   </property>
  </action>
  <action name="actionNode_s_Coordinates">
   <property name="text">
    <string>Node's Coordinates</string>
   </property>
   <property name="toolTip">
    <string>结点坐标</string>
   </property>
  </action>
  <action name="actionEdge_s_Weight">
   <property name="text">
    <string>Edge's Weight</string>
   </property>
   <property name="toolTip">
    <string>边的权重</string>
   </property>
  </action>
  <action name="actionRedigraph_s_Degrees">
   <property name="text">
    <string>Redigraph's Degrees</string>
   </property>
   <property name="toolTip">
    <string>结点度</string>
   </property>
  </action>
  <action name="actionOut_degree">
   <property name="text">
    <string>Out-degree</string>
   </property>
   <property name="toolTip">
    <string>出度</string>
   </property>
  </action>
  <action name="actionIn_degree">
   <property name="text">
    <string>In-degree</string>
   </property>
   <property name="toolTip">
    <string>入度</string>
   </property>
  </action>
  <action name="actionEasy_Pathway">
   <property name="text">
    <string>Easy Pathway</string>
   </property>
   <property name="toolTip">
    <string>简单通路</string>
   </property>
  </action>
  <action name="actionComplicated_Pathway">
   <property name="text">
    <string>Complicated Pathway</string>
   </property>
   <property name="toolTip">
    <string>复杂通路</string>
   </property>
  </action>
  <action name="actionEasy_Loop">
   <property name="text">
    <string>Easy Loop</string>
   </property>
   <property name="toolTip">
    <string>简单回路</string>
   </property>
  </action>
  <action name="actionComplicated_Loop">
   <property name="text">
    <string>Complicated Loop</string>
   </property>
   <property name="toolTip">
    <string>复杂回路</string>
   </property>
  </action>
  <action name="actionIncidence_Matrix">
   <property name="text">
    <string>Incidence Matrix</string>
   </property>
   <property name="toolTip">
    <string>关联矩阵</string>
   </property>
  </action>
  <action name="actionAdjacent_Matrix">
   <property name="text">
    <string>Adjacent Matrix</string>
   </property>
   <property name="toolTip">
    <string>邻接矩阵</string>
   </property>
  </action>
  <action name="actionBigraph">
   <property name="text">
    <string>Bigraph</string>
   </property>
   <property name="toolTip">
    <string>二部图</string>
   </property>
  </action>
  <action name="actionShortest_Path">
   <property name="text">
    <string>Shortest Path</string>
   </property>
   <property name="toolTip">
    <string>最短路径</string>
   </property>
  </action>
  <action name="actionCritical_Path">
   <property name="text">
    <string>Critical Path</string>
   </property>
  </action>
  <action name="actionFind_Euler_Loop">
   <property name="text">
    <string>Find Euler Loop</string>
   </property>
   <property name="toolTip">
    <string>寻找欧拉回路</string>
   </property>
  </action>
  <action name="actionFind_Hamiltonian_Loop">
   <property name="text">
    <string>Find Hamiltonian Loop</string>
   </property>
   <property name="toolTip">
    <string>寻找哈密顿回路</string>
   </property>
  </action>
  <action name="actionGraph_Coloring_Problem">
   <property name="text">
    <string>Graph Coloring Problem</string>
   </property>
   <property name="toolTip">
    <string>着色问题</string>
   </property>
  </action>
  <action name="actionDigraph_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Digraph Mode</string>
   </property>
   <property name="toolTip">
    <string>有向图模式</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionRedigraph_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Redigraph Mode</string>
   </property>
   <property name="toolTip">
    <string>无向图模式</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionTree_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tree Mode</string>
   </property>
   <property name="toolTip">
    <string>树模式</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionMinimise">
   <property name="text">
    <string>Minimise</string>
   </property>
   <property name="toolTip">
    <string>窗口最小化</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Maximize</string>
   </property>
   <property name="toolTip">
    <string>窗口最大化</string>
   </property>
  </action>
  <action name="actionMini_Mode">
   <property name="text">
    <string>Mini-Mode</string>
   </property>
   <property name="toolTip">
    <string>迷你模式</string>
   </property>
  </action>
  <action name="actionHide_ToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide ToolBar</string>
   </property>
   <property name="toolTip">
    <string>隐藏工具栏</string>
   </property>
  </action>
  <action name="actionConnect_Author">
   <property name="text">
    <string>Connect Author</string>
   </property>
   <property name="toolTip">
    <string>联系作者</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="text">
    <string>Welcome</string>
   </property>
   <property name="toolTip">
    <string>欢迎</string>
   </property>
  </action>
  <action name="actionHelp_Document">
   <property name="text">
    <string>Help Document</string>
   </property>
   <property name="toolTip">
    <string>帮助文档</string>
   </property>
  </action>
  <action name="actionFlie_List">
   <property name="text">
    <string>Flie List</string>
   </property>
   <property name="toolTip">
    <string>最近文件</string>
   </property>
  </action>
  <action name="actionPen_Thickness">
   <property name="text">
    <string>Pen Thickness</string>
   </property>
   <property name="toolTip">
    <string>画笔粗细</string>
   </property>
  </action>
  <action name="actionPen_Color">
   <property name="text">
    <string>Pen Color</string>
   </property>
   <property name="toolTip">
    <string>画笔颜色</string>
   </property>
  </action>
  <action name="actionBackground_Color">
   <property name="text">
    <string>Background Color</string>
   </property>
   <property name="toolTip">
    <string>背景色</string>
   </property>
  </action>
  <action name="actionArc">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/ARC.BMP</normaloff>:/icons/images/ARC.BMP</iconset>
   </property>
   <property name="text">
    <string>Arc</string>
   </property>
   <property name="toolTip">
    <string>弧线</string>
   </property>
  </action>
  <action name="actionStraight_Line">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/LINE.BMP</normaloff>:/icons/images/LINE.BMP</iconset>
   </property>
   <property name="text">
    <string>Straight Line</string>
   </property>
   <property name="toolTip">
    <string>直线</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/ELLIPSE.BMP</normaloff>:/icons/images/ELLIPSE.BMP</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
   <property name="toolTip">
    <string>椭圆</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/images/RECTANGL.BMP</normaloff>:/icons/images/RECTANGL.BMP</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionSetEnglish">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionSetChinese">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chinese</string>
   </property>
  </action>
  <action name="actionAdd_Annotation">
   <property name="icon">
    <iconset>
     <normalon>:/icons/images/800.bmp</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add Annotation</string>
   </property>
   <property name="toolTip">
    <string>添加注释</string>
   </property>
  </action>
  <action name="actionClues_Color">
   <property name="text">
    <string>Clues Color</string>
   </property>
  </action>
  <action name="actionClues_Thickness">
   <property name="text">
    <string>Clues Thickness</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionHide_ToolBar</sender>
   <signal>triggered()</signal>
   <receiver>general_utility_toolBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHide_ToolBar</sender>
   <signal>triggered()</signal>
   <receiver>graphical_toolBar</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinimise</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaximize</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>555</x>
     <y>370</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
